doc/sphinx/source/project/about.rst

```diff
@@ -1,108 +1,224 @@
+**platform-neutral** :abbr:`Intermediary Representation (IR)` to construct a 
+"stateless" graph. Each node, or *op*, in the graph corresponds to one 
+:term:`step` in a computation, where each step produces zero or more tensor 
+outputs from zero or more tensor inputs. 
 
 
-.. _framework-flexibility:
+How do I connect a framework? 
+-----------------------------
 
-How do I connect it to a framework? 
-------------------------------------
+The nGraph Library manages framework bridges for some of the more widely-known 
+frameworks. A bridge acts as an intermediary between the nGraph core and the 
+framework, and the result is a function that can be compiled from a framework. 
+A fully-compiled function that makes use of bridge code thus becomes a "function
+graph", or what we sometimes call an **nGraph graph**.  
 
-Currently, we offer *framework bridges* for some of the more widely-known 
-:doc:`frameworks <../framework-integration-guides>`. The bridge acts as an 
-intermediary between the *ngraph core* and the framework, providing a means 
-to use various execution platforms. The result is a function that can be 
-executed from the framework bridge.
+.. note:: Low-level nGraph APIs are not accessible *dynamically* via bridge code;
+   this is the nature of stateless graphs. However, do note that a graph with a 
+   "saved" checkpoint can be "continued" to run from a previously-applied 
+   checkpoint, or it can loaded as static graph for further inspection.
 
-Given that we have no way to predict how many more frameworks might be invented
-for either model or framework-specific purposes, it would be nearly impossible 
-for us to create bridges for every framework that currently exists (or that will 
-exist in the future). Thus, the library provides a way for developers to write 
-or contribute "bridge code" for various frameworks.  We welcome such 
-contributions from the DL community.
+For a more detailed dive into how custom bridge code can be implemented, see our 
+documentation on how to :doc:`../howto/execute`. To learn how TensorFlow and 
+MXNet currently make use of custom bridge code, see the section on 
+:doc:`../framework-integration-guides`.
 
+.. figure:: ../graphics/bridge-to-graph-compiler.png
+    :width: 733px
+    :alt: Compiling a computation
 
-How do I connect a DL training or inference model to nGraph?
--------------------------------------------------------------
+    JiT Compiling for computation
+
+Given that we have no way to predict how many other frameworks designed around 
+model, workload, or framework-specific purposes there may be, it would be  
+impossible for us to create bridges for every framework that currently exists 
+(or that will exist in the future). Thus, we provide documentation to help 
+developers and engineers work with a lower-level API that can be used to load 
+the function graph, to run Interpreter mode to help in writing "bridge code" for 
+new or novel frameworks or algorithms. 
+
```

Comment:
How about something like `Although we only support a few frameworks, we provide documentation to help developers and engineers`

Reply:
Fixed.